libraries/chain/vesting_balance_evaluator.cpp

```diff
@@ -26,8 +26,19 @@
 #include <graphene/chain/database.hpp>
 #include <graphene/chain/vesting_balance_evaluator.hpp>
 #include <graphene/chain/vesting_balance_object.hpp>
+#include <graphene/chain/hardfork.hpp>
 
 namespace graphene { namespace chain {
+namespace detail {
+   void check_vesting_balance_policy_hf_1268(const fc::time_point_sec& block_time, const vesting_policy_initializer& policy)
+   {
```

Comment:
Similar to `asset_evaluator.cpp` please add `TODO` comments to remove code after hardfork. Not needed in `proposal_evaluator.cpp` as the code is part of a struct that already have the todo comment. thank you again.

Reply:
Done.